src/helpers/ConfigHelper.ts

```diff
@@ -96,6 +96,9 @@ export class ConfigHelper {
     const mergedUserConfig = this.getUserConfigForConfigIntegrationKind(
       configIntegrationKind, userConfig, serverConfig, isUsingSubscriptionWorkaround);
 
+    const receiveReceiptsEnable = typeof serverConfig.features.receive_receipts_enable !== "undefined" ?
```

Comment:
Per our discussion: the above requested change would always return `true` so that wouldn't work

In either case, we decided to default the feature flag to OFF so backend can scale